snapcraft.yaml

```diff
@@ -1634,10 +1634,7 @@ parts:
       rm -rf "${CRAFT_PRIME}/lib/systemd/"
       rm -rf "${CRAFT_PRIME}/lib/udev/"
       rm -rf "${CRAFT_PRIME}/usr/local/"
-<<<<<<< HEAD
-=======
       rm -rf "${CRAFT_PRIME}/usr/share/"
->>>>>>> e436b6f2 (snapcraft: don't prime anything from the nasm part)
```

Comment:
would be better to not add them in the first place rather than remove them via later commit